src/index.html

```diff
@@ -22,6 +22,401 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#index.html">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="header__image"
+        />
+      </a>
+
+      <nav class="header__navigation">
+        <ul class="header__navigation-list">
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
```

Comment:
The `href` attribute is set to `#index.html`. Verify if this is the intended behavior. If these links should navigate to different sections or pages, update the `href` attributes accordingly.